examples/spring/src/main/resources/application.yml

```diff
@@ -2,3 +2,7 @@ graphql:
   subscriptions:
     # Send a ka message every 1000 ms (1 second)
     keepAliveInterval: 1000
+
+  playground:
+    enabled: true
+    endpoint: "/"
```

Comment:
We would also need to set the urls in the Playground html file. Not sure how we do this dynamically the best way

Reply:
Ahh `graphql-spqr` solves this issue like so

https://github.com/leangen/graphql-spqr-spring-boot-starter/blob/787041aff9ab1a87164f034ecd443e638f0febfb/graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/web/GuiController.java#L24-L28

Reply:
@smyrick 
Ahh, sorry I forgot to add that.
copy -> paste -> done :D

Reply:
lets change the default to `/playground` to avoid potential conflicts

Reply:
alright @dariuszkuc 
wait

Reply:
Right now implementing (in progress) Graphql and Altair.. should I merge with this branch..
@dariuszkuc @smyrick 

```
graphql:
  ide:
    - playground:
        enabled: true
        endpoin: "/playground"
    - graphiql:
        enabled: true
        endpoin: "/graphiql"
    - altair:
        enabled: true
        endpoin: "/altair"
```

Reply:
No need for `graphiql` or `altair`. Lets be opinionated and include just `playground`. Can you fix the Javadocs on the `PlaygroundConfigurationProperties` (currently javadoc is specified on `enabled` instead of `endpoint`)?

Reply:
alright then.
oke wait